src/main/kotlin/no/nav/syfo/service/OppgaveService.kt

```diff
@@ -42,12 +43,11 @@ class OppgaveService(
         }
     }
 
-    suspend fun duplikatOppgave(
+    suspend fun hentOppgave(
         journalpostId: String,
-        trackingId: String,
-        loggingMeta: LoggingMeta
-    ): Boolean {
-        return oppgaveClient.duplikatOppgave(journalpostId, trackingId, loggingMeta)
+        sykmeldingId: String
+    ): OppgaveResponse {
+        return oppgaveClient.hentOppgave(oppgavetype = "JFR", journalpostId = journalpostId, msgId = sykmeldingId)
```

Comment:
Her henter vi kun journalføringsoppgaver. Hva med fordelingsoppgaver?

Reply:
Ser nå at duplikatsjekken for fordelingsoppgaver ble dekket av skalBehandleJournalpost-metoden som ikke lenger sjekker om oppgaven finnes. Her vil vi potensielt opprette mange oppgaver hvis appen får en feil og restarter :/

Reply:
Kan kanskje håndtere fordelingsoppgave-dupliaktsjekk i den servicen som håndterer at fnr mangler (husker ikke hva den heter)? :)

Reply:
Er vi interessert i fordelingsoppgaver her? Vi regner vel ikke dem som duplikatoppgaver som kan brukes for å trigge smreg? 

Koden som sjekket for duplikater, som i praksis erstattes av denne, hentet også bare ut journalføringsoppgaver.

Reply:
Hmm, hvorfor tenker du at dette blir annerledes nå? Vi oppretter bare oppgaver for utenlandske sykmeldinger i papirmottaket, og for utenlandske sykmeldinger håndteres duplikatsjekk manuelt i opprettOppgave-funksjonen